### ISSUE TITLE
AddIdentity vs. AddDefaultIdentity vs. AddIdentityCore 

### ISSUE THREAD

**Issue by sbsw** [aspnet/Docs#7633]
These pages sometimes say we should use `services.AddDefaultIdentity`, and sometimes  `services.AddIdentity`.  Reading between the lines, AddIdentity exposes additional options for configuring Identity.  But there is no coherent description of the difference between these two methods.  We really need a section early in this documentation that presents these two registration methods and information about the follow-on fluent methods that are required and/or optional.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 804e24dd-7e30-8cf5-2c99-1ffacb4582f6
* Version Independent ID: 3e25547b-e2c9-6634-2c38-2a77207f17d4
* Content: [Scaffold Identity in ASP.NET Core projects](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-2.1&tabs=visual-studio)
* Content Source: [aspnetcore/security/authentication/scaffold-identity.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/security/authentication/scaffold-identity.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@sbsw  thanks for pointing this out. See #6367
Closing as a duplicate of #6367.

**Comment by Rick-Anderson** [comment]
@sbsw thanks for this. I've moved it to  #8434 and plan on providing a link to the new section in this doc. Would that work for you?

**Comment by sbsw** [comment]
Thanks @Rick-Anderson that would be great.  Since I wrote this issue, I've stumbled across various conflicting examples that purport to show the calls one would need with AddIdentity to replicate AddDefaultIdentity.  In any case, it's a nontrivial amount of code, none of which is clearly explained in the code examples.

As an aside, I want to give you guys kudos for the great documentation, with coherent explanations and great examples.  In particular, I appreciate the highlighted code in the examples that ties in with the discussion.
